lock-api/src/main/java/com/palantir/lock/client/RemoteTimelockServiceAdapter.java

```diff
@@ -32,10 +36,12 @@
 public final class RemoteTimelockServiceAdapter implements TimelockService {
```

Comment:
I've found the logic here pretty hard to follow to be honest, haven't really gotten a good idea of what it should look like just yet, but yeah.